torch/testing/_internal/common_modules.py

```diff
@@ -315,12 +356,42 @@ def reference_fn(m, p, i, t, constructor_kwargs=constructor_kwargs):
 
         module_inputs.append(
             ModuleInput(constructor_input=FunctionInput(**constructor_kwargs),
-                        forward_input=FunctionInput(make_input((15, 10)),
-                                                    torch.empty(15, device=device).uniform_().mul(10).floor().long()),
+                        forward_input=FunctionInput(make_input((15, 4)),
+                                                    torch.empty(15, device=device).uniform_().mul(4).floor().long()),
```

Comment:
Wanted to decrease the test time since there are quite a lot of sample inputs, I don't think 10 is a special number here so any other number of classes is probably also good :)